src/phoenix/server/main.py

```diff
@@ -290,12 +292,6 @@ def _get_pid_file() -> Path:
         force_fixture_ingestion = args.force_fixture_ingestion
         scaffold_datasets = args.scaffold_datasets
     host: Optional[str] = args.host or get_env_host()
-    display_host = host or "localhost"
-    # If the host is "::", the convention is to bind to all interfaces. However, uvicorn
-    # does not support this directly unless the host is set to None.
-    if host and ":" in host:
-        # format IPv6 hosts in brackets
-        display_host = f"[{host}]"
```

Comment:
Not clear what we're removing this for?

Reply:
It's not used

Reply:
When I moved the code into the `main()` function, the IDE yelled at me for it